o4/o4.py

```diff
@@ -1159,6 +1159,13 @@ def move_except(from_dir, to_dir, but_not):
 
     target = os.getcwd()
     source = f'{target}/.o4/cleaning'
+    cleaned = f'{target}/.o4/cleaned'
+    if rm_empty_dirs(cleaned):
+        print(f'*** ERROR: Unhandled files still exist from a previous clean run.')
+        print(f'           Please delete (if unwanted) or move files from {cleaned}')
+        print(f'           back to its place in {target}.')
+        sys.exit(1)
+
```

Comment:
I prefer this to renaming an existing "cleaned" -- possibly more than once. Really, if the clean left detritus, it should be taken care of by the user.

Reply:
Agreed. This is very elegant. 
Is there a -f option?

Reply:
You really want one? I'm thinking that could be too easy for people to do, just to get things going. Kind of "-f fatigue".

Reply:
Better not then. We can always add it if we change our minds.